llvm/docs/HowToSubmitABug.rst

```diff
@@ -112,15 +112,20 @@ If this does crash, then you should be able to debug this with the following
 Run this, then file a bug with the instructions and reduced .bc
 files that bugpoint emits.
 
-If bugpoint doesn't reproduce the crash, ``llvm-reduce`` is an alternative
-way to reduce LLVM IR. Create a script that repros the crash and run:
+If bugpoint doesn't reproduce the crash,
+:doc:`llvm-reduce <CommandGuide/llvm-reduce>` is an alternative way to reduce
+LLVM IR. Create a script that repros the crash and run:
 
 .. code-block:: bash
 
    llvm-reduce --test=path/to/script foo.bc
 
-which should produce reduced IR that reproduces the crash. Be warned the
-``llvm-reduce`` is still fairly immature and may crash.
+which should produce reduced IR that reproduces the crash.
+
+.. TIP::
+   ``llvm-reduce`` is still fairly immature and may crash. On the other hand,
+   unlike ``bugpoint``, ``llvm-reduce`` is multi-threaded and can therefore
```

Comment:
should mention that you need to pass `-j $NUM_THREADS` to get the multithreading

Reply:
Good idea, done: https://github.com/llvm/llvm-project/pull/148904/commits/64e499dedfd4258c5a009c832eff422439f8bd8f